pyprof/nvtx/nvmarker.py

```diff
@@ -312,15 +208,28 @@ def wrapper_func(*args, **kwargs):
             # Pop trace marker
             nvtx.range_pop()
 
+        if config.capture_input_ops:
+            dlprof.capture_outputs(dlprof.call_id, result)
+            # Store the callid -> op_name mapping
+            if traceMarker_str is not "":
```

Comment:
Was there a case where this was true?
Is there anything later in the code that assumes the call_id will exist in dlprof.call_id_to_op_map?

Reply:
Resnet failed stating that traceMarker_str was accessed before defined.